embeddings/pipeline/lightning_classification.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Dict, Optional, Sequence, Union
+
+import datasets
+import pytorch_lightning as pl
+from numpy import typing as nptyping
+
+from embeddings.data.datamodule import TextClassificationDataModule
+from embeddings.data.io import T_path
+from embeddings.evaluator.text_classification_evaluator import TextClassificationEvaluator
+from embeddings.model.lightning_model import LightningModel
+from embeddings.pipeline.lightning_pipeline import LightningPipeline
+from embeddings.task.lightning_task.text_classification import TextClassification
+
+
+class LightningClassificationPipeline(
+    LightningPipeline[datasets.DatasetDict, Dict[str, nptyping.NDArray[Any]], Dict[str, Any]]
+):
+    DEFAULT_TASK_TRAIN_KWARGS = {"gpus": 1, "auto_select_gpus": True}
+    DEFAULT_TASK_MODEL_KWARGS = {"use_scheduler": True}
+
+    def __init__(
+        self,
+        embedding_name: str,
+        dataset_name: str,
+        input_column_name: Union[str, Sequence[str]],
+        target_column_name: str,
+        output_path: T_path,
+        tokenizer_name: Optional[str] = None,
+        load_dataset_kwargs: Optional[Dict[str, Any]] = None,
+        task_model_kwargs: Optional[Dict[str, Any]] = None,
+        task_train_kwargs: Optional[Dict[str, Any]] = None,
+    ):
+        datamodule = TextClassificationDataModule(
```

Comment:
Parameters of DataModule (batch_size, seq_length) are not set here, therefore can't be set

Reply:
fixed